File: src/test/kotlin/no/nav/helsearbeidsgiver/utils/env/EnvironmentTest.java
package no.nav.helsearbeidsgiver.utils.env;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class EnvironmentTest {

    @Test
    public void shouldRead() {
        DemoSettings object = Environment.read(DemoSettings.class);
        assertNotNull(object.javaHome);
    }

    @Test
    public void shouldInject() {
        DemoSettings object = new DemoSettings();
        Environment.inject(object);
        assertNotNull(object.javaHome);
    }

}

File: src/test/kotlin/no/nav/helsearbeidsgiver/utils/env/EnvironmentTest.kt
package no.nav.helsearbeidsgiver.utils.env

import io.kotest.core.spec.style.StringSpec
import io.kotest.matchers.string.shouldNotBeEmpty

class EnvironmentTest : StringSpec({

    "skal kunne lese inn miljø variabler" {
        val settings = DemoSettings()
        settings.javaHome.shouldNotBeEmpty()
    }
})
